src/openrct2/core/MemoryStream.cpp

```diff
@@ -18,17 +18,6 @@
 #include "Memory.hpp"
 #include "MemoryStream.h"
 
-MemoryStream::MemoryStream()
-{
-    _access = MEMORY_ACCESS_READ |
-              MEMORY_ACCESS_WRITE |
-              MEMORY_ACCESS_OWNER;
-    _dataCapacity = 0;
-    _dataSize = 0;
-    _data = nullptr;
-    _position = nullptr;
-}
-
 MemoryStream::MemoryStream(const MemoryStream &copy)
 {
     _access = copy._access;
```

Comment:
Sorry I didn't see that `_data` and `_position` were only set if `_access == MEMORY_ACCESS_OWNER`